src/schemas/manifest.schema.json

```diff
@@ -50,9 +50,8 @@
       }
     },
     "chain": {
-      "type": ["string", "object"],
-      "description": "Indicate that this DAppNode Package is a blockchain node so the admin UI shows its syncing status",
-      "properties": {
+      "oneOf": [{"type": "string", "enum": ["ethereum", "ethereum-beacon-chain", "ethereum2-beacon-chain-prysm", "bitcoin", "monero"]},
+       {"type": "object","properties": {
```

Comment:
Please format with proper indentation. Run this through VSCode JSON formater or any online tool for that